dashboard/src/pages/Capability/Components/ShowComponent/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+
+import { Modal, Space, Table, Tag, Tooltip, Typography } from 'antd';
+
+import { ShowParameters } from './types';
+
+export default ({ name, parameters }: ShowParameters) => {
+  const [localVisible, setLocalVisible] = useState(false);
+
+  const { Paragraph, Text } = Typography;
+
+  useEffect(() => {
+    setLocalVisible(name !== '');
+  }, [name, parameters]);
+
+  return (
+    <Modal
+      forceRender
+      visible={localVisible}
+      onCancel={() => setLocalVisible(false)}
+      footer={null}
+      width={1000}
+      maskClosable
+      destroyOnClose
+    >
+      <h2>{`Type: ${name}`}</h2>
+      <Table
+        columns={[
+          {
+            title: 'Name',
+            dataIndex: 'name',
+            key: 'name',
+            width: 200,
+            render: (text, row) => [
+              <Paragraph copyable={{ text: row.name }}>
```

Comment:
Please set a unique key.  
fix `Warning: Each child in a list should have a unique "key" prop.`